kubernetes/aks/system/oneuptime/base/ingress.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: oneuptime
+  annotations:
+    # https://oneuptime.com/docs/probe/ip-address
+    nginx.ingress.kubernetes.io/whitelist-source-range: 205.194.32.0/24,51.159.99.250,62.210.173.51,51.159.101.36,172.174.206.132,57.151.99.117
```

Comment:
I do wonder if alpha canada (digital services) maintains a list of GoC managed IP ranges somewhere?

should this be a secret?

Reply:
Except the first IP, all the IP's are from public endpoints of oneuptime services and are not ours.